zerver/lib/webhooks/common.py

```diff
@@ -20,5 +22,13 @@ def check_send_webhook_message(
     else:
         if user_specified_topic is not None:
             topic = user_specified_topic
-        check_send_stream_message(user_profile, request.client,
-                                  stream, topic, body)
+
+        try:
+            check_send_stream_message(user_profile, request.client,
+                                      stream, topic, body)
+        except StreamDoesNotExistError:
+            # A PM will be sent to the bot_owner by check_message, notifying
+            # that the webhook bot just tried to send a message to a non-existent
+            # stream, so we don't need to re-raise it since it clutters up
+            # webhook-errors.log
+            pass
```

Comment:
Do you think we want to still raise a `JsonableError` here so that the request doesn't return a 200 status to the server?  I'm not sure.  Otherwise this lgtm.

I guess if we wanted to do that, I'd want to also make a change to not log JsonableError exceptions here:

```
            try:                                                                                    
                return view_func(request, user_profile, *args, **kwargs)                            
            except Exception as err:                                                                
                log_exception_to_webhook_logger(request, user_profile)                              
                raise err                                                                           
```

which I'd do by adding a first exception block of the form
```
exception JsonableError as err:
    raise err
```

Reply:
This change concept is probably involved enough that it should be its own PR.

Reply:
@timabbott: Yeah, I think we should do that! I'll open a PR with that change. :) Thanks!